app/models/concerns/client_image_consumer.rb

```diff
@@ -84,6 +87,10 @@ def set_local_client_image_cache(image_data) # rubocop:disable Naming/AccessorMe
       client_files.window.tagged_with('Client Headshot').order(updated_at: :desc).limit(1)&.first
     end
 
+    private def source_local_image
```

Comment:
I was also wondering whether this should be made available as an attribute on `GrdaWarehouse::Hud::Client`?